.eslintrc

```diff
@@ -75,5 +76,18 @@
         "node": true,
         "browser": true
     },
+    "overrides": [{
+        "files": ["katex.js", "src/**/*.js"],
+        "rules": {
+            "import/no-unresolved": 2,
```

Comment:
Which files wouldn't we want this enabled for?

Reply:
@kevinbarabash Files which run on Node and are not included in the build.